### PR TITLE
Ricebox

### LINKED ISSUE #38: add go-rice
we're doing this the HARD way :)
add go-rice to have simple dev and production deploys!

### PR DESCRIPTION
**Pull request by bketelsen**
This PR adds go-rice for stable file resolution in production and development environments.
It also modifies the builds and deployments accordingly for Kubernetes.

fixes #38

### CODE REVIEW

File: actions/render.go [hunk 1]
```diff
@@ -2,36 +2,26 @@ package actions
 
 import (
 	"net/http"
-	"path"
-	"runtime"
 
+	rice "github.com/GeertJohan/go.rice"
```
Comment (briandowns) [inline actions/render.go:6]: Is this import alias necessary?

**Review by briandowns — APPROVED** [review]

File: actions/app.go [hunk 2]
```diff
@@ -29,32 +25,28 @@ func App() http.Handler {
...
 	a.GET("/home", HomeHandler)
+	a.GET("/", CountdownHandler)
 	adm := a.Group("/admin")
 	adm.GET("/index", AdminHandler)
 
 	return a
 }
 
-func setTemplate() buffalo.MiddlewareFunc {
+func setLayout() buffalo.MiddlewareFunc {
```
Comment (markbates) [inline actions/app.go:39]: @bketelsen you have a  concurrency issue here. it's possible for two requests to come in at the same time and override the other one's preferred layout. you should use a separate renderer for the admin stuff.

**Comment by bketelsen** [comment]
Damn.  I had that originally.  I took it out because I was fighting with referencing the HTMLLayout from within the ricebox.  I'll try again in a new PR

**Comment by bketelsen** [comment]
tracking with : 
https://github.com/gopheracademy/gcon/issues/46

**Pull request merged**